admin/src/main/java/org/bahmni/module/admin/csv/service/CSVPatientService.java

```diff
@@ -56,7 +56,11 @@ public Patient save(PatientRow patientRow) throws ParseException {
             patient.setBirthdateFromAge(Integer.parseInt(patientRow.age), new Date());
         }
         patient.setGender(patientRow.gender);
-        patient.addIdentifier(new PatientIdentifier(patientRow.registrationNumber, getPatientIdentifierType(), null));
+        PatientIdentifier patientIdentifier = new PatientIdentifier(patientRow.registrationNumber, getPatientIdentifierType(), null);
+        patientIdentifier.setPreferred(true);
+        patient.addIdentifier(patientIdentifier);
+
+        addExtraPersonIdentifiers(patient, patientRow);
```

Comment:
@supriyatw @tarunkumar-tw i think this can be "addExtraPatientIdentifiers" instead of "addExtraPersonIdentifiers" ?